functions/check-expiry/v2.js

```diff
@@ -9,6 +9,9 @@ const v2 = {
     // change the check status
     const checkData = await expireChecks(logger)
 
+    const checkIDs = checkData.map(e => e.checkId)
+    logger(`check-expiry: Updated ${checkData.length} checks' status to \`EXP\`: ${checkIDs.join(', ')}`)
```

Comment:
I wonder if using checkCode instead would make it easier to search application insights?